_overviews/scala3-book/packaging-imports.md

```diff
@@ -211,7 +211,7 @@ package foo
 
 class ClassA:
   import scala.util.Random   // inside ClassA
-  def printRandom {
+  def printRandom:
```

Comment:
```suggestion
  def printRandom(): Unit =
```

Reply:
Looks like line 202 needs this change as well.